src/vstd_ext/map_lib.rs

```diff
@@ -25,4 +25,36 @@ pub proof fn union_prefer_right_self_changes_nothing<K, V>()
             map.union_prefer_right(map) =~= map,
 {}
 
+pub proof fn lemma_values_finite<K, V>(m: Map<K, V>)
```

Comment:
Can you leave a TODO saying that we will use the Verus native lemma_values_finite once we update to the most recent Verus?

Reply:
Sure. This lemma was just added for my use